packages/client/components/ActivityLibrary/ActivityDetails/MeetingDetails.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
+1 This is effectively the same as `TemplateDetails` with some tweaks, right? Why not merge the two components and do conditional renderings if the activity isn't a template? That would ensure consistency between generic activity details for template activities and generic activity details for non-template activities. I.e. if we changed anything about how generic details are rendered, we'd need to change that both here and in `TemplateDetails`, instead of in just one place.